tests/test_security_curve.cpp

```diff
@@ -27,6 +27,21 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+// TODO remove this workaround for handling libsodium/tweetnacl
+
+#if defined ZMQ_CUSTOM_PLATFORM_HPP
+#include "platform.hpp"
+#else
+#include "../src/platform.hpp"
+#endif
+
+#ifndef ZMQ_USE_TWEETNACL
+#define ZMQ_USE_TWEETNACL
+#endif
+#ifdef ZMQ_USE_LIBSODIUM
+#undef ZMQ_USE_LIBSODIUM
```

Comment:
If I follow correctly, this means the unit tests are effectively not using the new memory allocators, right?

Reply:
Well they are on the side of the tested libzmq functions. Just not for the mock curve client messages

Reply:
Ah yes of course - thanks!